package.json

```diff
@@ -125,7 +125,7 @@
     "ovh-angular-tail-logs": "ovh-ux/ovh-angular-tail-logs#^1.1.2",
     "ovh-angular-toaster": "ovh-ux/ovh-angular-toaster#^0.8.0",
     "ovh-angular-user-pref": "ovh-ux/ovh-angular-user-pref#^0.3.1",
-    "ovh-api-services": "ovh-ux/ovh-api-services#^3.9.0",
+    "ovh-api-services": "ovh-ux/ovh-api-services#^3.15.1",
```

Comment:
By upgrading the `ovh-api-services` dependency, `yarn.lock` file must be committed as well.